community/fulltext-index/src/main/java/org/neo4j/kernel/api/impl/fulltext/FulltextAdapter.java

```diff
@@ -35,8 +36,14 @@
 {
     SchemaDescriptor schemaFor( EntityType type, String[] entityTokens, Properties indexConfiguration, String... properties );
 
+    SchemaDescriptor schemaForSort( EntityType type, String[] entityTokens, Properties indexConfiguration, String[] properties, String[] sortProperties,
```

Comment:
The name of this method doesn't seem to match the semantic meaning of `schemaFor`.
Something like `schemaForSortOn` or `sortSchemaFor` might be a more consistent name.

Following the convention of `schemaFor`, `schemaForSort` implies that it's creating a schema for a `sort` entity type, rather than creating a schema for sorting on a node or relationship.

Reply:
I like how you're thinking here and wondering if it should be `schemaSortFor`. That's my vote.

Reply:
Good catch, I will make a refactor for `schemaSortFor`